Node2Code_Engine/Query/InlineExpression.cs

```diff
@@ -0,0 +1,136 @@
+                else
+                    expression = SyntaxFactory.IdentifierName(data.Data.ToString());
+
+                if (expression != null)
+                {
+                    newVariables[data.BHoM_Guid] = new Variable
+                    {
+                        Expression = expression,
+                        Type = data.DataType,
+                        SourceId = data.BHoM_Guid
+                    };
+                }
+            }
+
+            return newVariables;
+        }
+
+        /***************************************************/
+
+        public static Dictionary<Guid, Variable> InlineExpression(this GetPropertyNode node, Dictionary<Guid, Variable> variables)
+        {
+            List<ExpressionSyntax> arguments = node.Inputs.Select(x => Query.ArgumentValue(x, variables)).ToList();
+
+            Guid id = node.Outputs.First().BHoM_Guid;
+
+            Variable variable = new Variable
+            {
+                Expression = SyntaxFactory.MemberAccessExpression(
+                    SyntaxKind.SimpleMemberAccessExpression,
+                    arguments[0],
+                    SyntaxFactory.IdentifierName(arguments[1].ToFullString().Replace("\"", ""))
+                ),
+                SourceId = id,
+                Type = node.Outputs.First().DataType
+            };
+
+            return new Dictionary<Guid, Variable> {
+                { id, variable }
+            };
+        }
+
+        /***************************************************/
+
+        public static Dictionary<Guid, Variable> InlineExpression(this ExplodeNode node, Dictionary<Guid, Variable> variables)
+        {
+            List<ExpressionSyntax> arguments = node.Inputs.Select(x => Query.ArgumentValue(x, variables)).ToList();
+
+            IEnumerable<DataParam> validOutputs = node.Outputs.Where(x => x.TargetIds.Count > 0);
+            if (validOutputs.Count() == 0)
+                return new Dictionary<Guid, Variable>();
+            else
+                return validOutputs.ToDictionary(
+                    output =>   output.BHoM_Guid,
+                    output => new Variable
+                    {
+                        Expression = SyntaxFactory.MemberAccessExpression(
+                            SyntaxKind.SimpleMemberAccessExpression,
+                            arguments[0],
+                            SyntaxFactory.IdentifierName(output.Name)
+                        ) as ExpressionSyntax,
+                        SourceId = output.BHoM_Guid,
+                        Type = output.DataType
+                    }
+                );
+        }
+
+
+        /***************************************************/
+        /**** Private Methods                           ****/
+        /***************************************************/
+
+        public static Dictionary<Guid, Variable> InlineExpression(this INode node, Dictionary<Guid, Variable> variables)
```

Comment:
Comment says private but method says public. If meant to be public, then - Documentation?